doc/source/getting_started/launcher.rst

```diff
@@ -0,0 +1,228 @@
+address of that instance, so you can connect to it.
+For example, if on your local network at IP address ``192.168.0.1`` there is a
+computer running MAPDL on the port 50052, you can connect to it with:
+
+.. code:: pycon
+
+    >>> mapdl = Mapdl("192.168.0.1", port=50052)
+
+Alternatively you can use a hostname:
+
+.. code:: pycon
+
+    >>> mapdl = Mapdl("myremotemachine", port=50052)
+
+Note that you must have started an MAPDL instance in gRPC mode on the computer with
+the mentioned IP address/hostname for this to work, since PyMAPDL cannot launch remote instances.
+
+
+Setting MAPDL location in PyMAPDL
+---------------------------------
+
+To run PyMAPDL must know the location of the MAPDL binary. 
+Most of the time this can be automatically determined, but
+the location of MAPDL must be provided for non-standard installations.
+When running for the first time, PyMAPDL requests the
+location of the MAPDL executable if it cannot automatically find it.
+
+You can test your installation of PyMAPDL and set it up by running
+the :func:`launch_mapdl() <ansys.mapdl.core.launcher.launch_mapdl>` function:
+
+.. code:: python
+
+    from ansys.mapdl.core import launch_mapdl
+
+    mapdl = launch_mapdl()
+
+Python automatically attempts to detect your MAPDL binary based on
+environmental variables.
+You can specify MAPDL installation using one of the two following environment
+variables:
+
+* ``AWP_ROOTXXX`` where ``XXX``` is the three digits version. This environment variable
+  contains the path of the Ansys installation which version match ``XXX``.
+  For instance, ``AWP_ROOT222=/ansys_inc`` contains the path of an Ansys 2022R2 installation.
+
+* ``PYMAPDL_MAPDL_EXEC`` contains the path the Ansys MAPDL executable.
+  For instance, ``PYMAPDL_MAPDL_EXEC=/ansys_inc/v222/ansys/bin/ansys222``.
+
+If PyMAPDL is unable to find a copy of MAPDL, you
+are prompted for the location of the MAPDL executable.
+
+Here is a sample input for Linux:
+
+.. code:: output
+
+    Enter location of MAPDL executable: /usr/ansys_inc/v222/ansys/bin/ansys222
+
+Here is a sample input for Windows:
+
+.. code:: output
+
+    Enter location of MAPDL executable: C:\Program Files\ANSYS Inc\v222\ANSYS\bin\winx64\ansys222.exe
+
+The settings file is stored locally, which means that you are not prompted
+to enter the path again. If you must change the default Ansys path
+(meaning change the default version of MAPDL), run the following:
+
+.. code:: python
+
+    from ansys.mapdl import core as pymapdl
+
+    new_path = "C:\\Program Files\\ANSYS Inc\\v212\\ANSYS\\bin\\winx64\\ansys222.exe"
+    pymapdl.change_default_ansys_path(new_path)
+
+Also see the :func:`change_default_ansys_path() <ansys.mapdl.core.change_default_ansys_path>` method and
+the :func:`find_ansys() <ansys.mapdl.core.find_ansys>` method.
+
+Additionally, it is possible to specify the executable in each PyMAPDL script using the keyword argument ``exec_file``. 
```

Comment:
```suggestion
Additionally, it is possible to specify the executable in each PyMAPDL script using the ``exec_file`` keyword argument. 
```
It's always better to follow a tagged code entity with the noun describing what the object type is.